src/main/java/com/example/demo/db/repo/AvailableShiftsRepo.java

```diff
@@ -0,0 +1,45 @@
+package com.example.demo.db.repo;
+
+import com.example.demo.db.entities.AvailableShifts;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.Collection;
+import java.util.List;
+
+public interface AvailableShiftsRepo extends CrudRepository<AvailableShifts,Integer> {
```

Comment:
Please read [this](https://github.com/Neuberger-Shlomo/java-project-20503/pull/7#discussion_r1160299027)
It will help you to shorten your code...
and also add JavaDoc format docs